type_classes.md

```diff
@@ -597,6 +932,19 @@ def ite {α : Sort u} (c : Prop) [h : Decidable c] (t e : α) : α :=
 # end Hidden
 ```
 
+So the following are equivalent:
+```lean
+#eval if 3 > 4 then 10 else 20        -- 20
+#eval ite (3 > 4) 10 20               -- 20
+```
+
+Notice in Lean an expression (3 > 4) can simply be passed as a
+parameter to the function `ite` because the parameter declaration
+`(c: Prop)` says `c` is of type `Prop`.
+
+BUGBUG: is this unique to Lean or Lean and haskell, you certainly
```

Comment:
What exactly are you referring to? I can certainly write python code like `ite((3 > 4), 10, 20)`.